### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by d-atkins**
- Add stat tracker which takes a hash of file paths and passes them to respective collections
- Add game, game_teams, and tests
- Add game_collection, game_teams_collection, and tests
- Set up simplecov

### CODE REVIEW

File: test/stat_tracker_test.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+require './test/test_helper'
+require './lib/stat_tracker'
+
+class StatTrackerTest < Minitest::Test
```
Comment (tylertomlinson) [inline test/stat_tracker_test.rb:4]: StatTracker test doesn't seem to be working. Need to revisit

File: test/game_teams_collection_test.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  def test_it_has_attributes
+    assert_instance_of Array, @game_teams_collection.game_teams_array
+  end
+
+  def test_it_can_create_games_from_csv
+    assert_instance_of GameTeams, @game_teams
+    assert_equal "away", @game_teams.hoa
+    assert_equal "LOSS", @game_teams.result
```
Comment (Yetidancer) [inline test/game_teams_collection_test.rb:21]: HOA and result should be all we need for iteration 2 stats

File: test/stat_tracker_test.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  end
+
+  def test_it_can_create_a_game_collection
+    assert_instance_of GameCollection, @stat_tracker.game_collection
+    assert_instance_of Array, @stat_tracker.game_collection.games
+  end
+
+  def test_it_can_create_a_team_collection
+    assert_instance_of TeamCollection, @stat_tracker.team_collection
+    assert_instance_of Array, @stat_tracker.team_collection.teams
```
Comment (mronauli) [inline test/stat_tracker_test.rb]: we haven't actually made a team collection, so this test isn't passing

File: test/game_teams_test.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+require './test/test_helper'
+require './lib/game_teams'
+
+class GameTeamsTest < Minitest::Test
+  def setup
+    @game_teams = GameTeams.new({hoa: "away", result: "WIN"})
```
Comment (Yetidancer) [inline test/game_teams_test.rb:6]: can populate with more data from csv if necessary down the line

File: test/game_teams_collection_test.rb (hunk 2 above)
Comment (d-atkins) [reply test/game_teams_collection_test.rb:21]: Noted

File: test/game_collection_test.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+  def test_it_has_attributes
+    assert_instance_of Array, @game_collection.games
+  end
+
+  def test_it_can_create_games
+    assert_instance_of Game, @game
+    assert_equal "20122013", @game.season
+    assert_equal 2, @game.away_goals
+    assert_equal 3, @game.home_goals
```
Comment (Yetidancer) [inline test/game_collection_test.rb:22]: beautifully written. hands of gold

File: test/game_test.rb [hunk 6]
```diff
@@ -0,0 +1,18 @@
+require './test/test_helper'
+require './lib/game'
+
+class GameTest < Minitest::Test
+  def setup
+    @game = Game.new({season: "20122013", away_goals: 3, home_goals: 4})
```
Comment (Yetidancer) [inline test/game_test.rb:6]: thats a dubya for the home team

File: test/stat_tracker_test.rb (hunk 3 above)
Comment (d-atkins) [reply test/stat_tracker_test.rb]: Pushed new changes

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1 @@
+**/coverage
```
Comment (Yetidancer) [inline .gitignore:1]: still dont understand this ish

**Review by tylertomlinson — COMMENTED** [review]
needs to be reviewed.

File: test/stat_tracker_test.rb (hunk 1 above)
Comment (d-atkins) [reply test/stat_tracker_test.rb:4]: Pushed changes

**Pull request merged**